src/main/java/bhs/devilbotz/RobotContainer.java

```diff
@@ -218,20 +219,21 @@ public Command getAutonomousCommand(AutonomousModes autoMode) {
               new ScoreAndMobility(arm, driveTrain, maxDistance, gripper));
           break;
         case DOCK_AND_ENGAGE:
-          autonomousCommand.addCommands(new DockAndEngage(driveTrain, maxDistance));
+          autonomousCommand.addCommands(new DockAndEngage(driveTrain, maxDistance, startAngle));
           break;
         case MOBILITY_DOCK_AND_ENGAGE_HUMAN_SIDE:
           autonomousCommand.addCommands(
               new MobilityDockAndEngage(
-                  driveTrain, CommunityLocation.HUMAN, DriverStation.getAlliance()));
+                  driveTrain, CommunityLocation.HUMAN, DriverStation.getAlliance(), startAngle));
           break;
         case MOBILITY_DOCK_AND_ENGAGE_WALL_SIDE:
           autonomousCommand.addCommands(
               new MobilityDockAndEngage(
-                  driveTrain, CommunityLocation.WALL, DriverStation.getAlliance()));
+                  driveTrain, CommunityLocation.WALL, DriverStation.getAlliance(), startAngle));
           break;
         case SCORE_DOCK_AND_ENGAGE:
-          autonomousCommand.addCommands(new ScoreDockAndEngage(arm, driveTrain, gripper));
+          autonomousCommand.addCommands(
+              new ScoreDockAndEngage(arm, driveTrain, gripper, startAngle));
           break;
         case SCORE_MOBILITY_DOCK_ENGAGE:
```

Comment:
It would be really nice if we could separate these into separate files, its starting to get quite confusing.

Reply:
I agree. I'll move the autonomous routine logic to a different class.

Reply:
Done.